treescript/src/treescript/versionmanip.py

```diff
@@ -19,27 +20,41 @@
     "config/milestone.txt",
     "mail/config/version.txt",
     "mail/config/version_display.txt",
-    "version.txt",  # Fenix
+    "version.txt",  # Github repositories
 )
 
 _VERSION_CLASS_PER_BEGINNING_OF_PATH = {
     "browser/": FirefoxVersion,
     "config/milestone.txt": GeckoVersion,
     "mail/": ThunderbirdVersion,
-    "version.txt": FenixVersion,
 }
 
+_VERSION_CLASS_PER_END_OF_SOURCE_REPO = {
+    "fenix": MobileVersion,
+    "focus-android": MobileVersion,
+    "android-components": MavenVersion,
+}
+
+
+def _find_what_version_parser_to_use(file_, source_repo):
+    version_classes = [cls for path, cls in _VERSION_CLASS_PER_BEGINNING_OF_PATH.items() if file_.startswith(path)]
+
+    number_of_version_classes = len(version_classes)
+    if number_of_version_classes > 0:
+        if number_of_version_classes > 1:
```

Comment:
Maybe reorder to avoid nesting the `if`s

Reply:
Sounds good, done!